R/clean_data.R

```diff
@@ -135,7 +135,9 @@ clean_data <- function(data, ...) {
   ## | Column names in 'keep' will not be modified.
   ## -----
   if (!is.null(params[["standardize_column_names"]])) {
-    message("\ncleaning column names\n")
+    cli::cli_alert_info(
+      tr_("\nCleaning column names")
```

Comment:
Can you double check if it's necessary to include `\n` for cli strings please? My initial assumption would be no

Reply:
This is actually not needed.